core/js/src/main/scala/scalaz/zio/App.scala

```diff
@@ -1,10 +1,22 @@
 package scalaz.zio
 
-trait App {
+trait App extends RTS {
+  sealed abstract class ExitStatus extends Serializable with Product
+  object ExitStatus extends Serializable {
```

Comment:
There's no need for `ExitStatus` on Javascript, I'd delete this.

Reply:
Deleted.